images/techdocs/context/Dockerfile

```diff
@@ -70,5 +70,17 @@ RUN pip install \
     --requirement requirements-techdocs.txt && \
     rm -rf requirements-techdocs.txt
 
+COPY maker /usr/local/bin/maker
+RUN \
```

Comment:
can we not do something smart and add all make targets here through a script instead of manual specify?

Reply:
Probably, but I not sure how to write a good and stable Makefile parser at the moment.